src/modules/px4iofirmware/mixer.cpp

```diff
@@ -167,9 +180,10 @@ mixer_tick(void)
 	 * FMU or from the mixer.
 	 *
 	 */
+	bool effective_safety_off = (r_status_flags & PX4IO_P_STATUS_FLAGS_SAFETY_OFF) != 0 || override_enabled;
```

Comment:
This would lead on PX4 to immediate arming in manual flight mode. I need to find something that does only bypass arming on reboots.